### ISSUE TITLE
[REQUEST] Flat Car Linux Google Compute Support

### ISSUE THREAD

**Issue by bobhenkel** [poseidon/typhoon#528]
### Environment

* Platform: google-cloud
* OS: flatcar-linux

### Feature

Can Typhoon be made to support Flat Car Linux on Google Compute? Assuming folks publish the image into their own google bucket per Flat Car instructions here [https://docs.flatcar-linux.org/os/booting-on-google-compute-engine/](https://docs.flatcar-linux.org/os/booting-on-google-compute-engine/)

Is there any big-ticket item that keeps it from being supported?

Some of the desire for this is not knowing what happens to CoreOS going forward. I know as of now all is nice, but who knows the future.  Flat Car Linux is at least a nice safety parachute to start exploring and testing with on my clusters.

**Comment by dghubble** [comment]
It is possible to build and publish images to Google Cloud on your own and use them if you're curious. In the past, I documented this process to provide Fedora Atomic (deprecated) support on GCP, despite Fedora not publishing official GCP images. I won't support that process again - explaining workarounds for vendor's not publishing images is a burden, folks get confused and perceive it as a project problem. 

When Kinvolk or Fedora publish Flatcar Linux or Fedora CoreOS images (respectively) for GCP at a regular cadence, I'll add in the corresponding options, docs, and/or implementations. Motivated folks are welcome to independently explore publishing the images on their own (check old docs) until the OS vendors have official images.

I do not have compatibility concerns. The main porting challenges (bare-metal vs cloud, new OS, etc) have been solved.
